addon/components/u-edit/shared-modules/drag-and-drop.js

```diff
@@ -0,0 +1,88 @@
+import Uploader from '@upfluence/ember-upf-utils/uploader';
+import Configuration from '@upfluence/ember-upf-utils/configuration';
+
+export default class ModuleBuilder {
+  constructor(session, toast) {
+    Object.assign(this, { session, toast });
+  }
+
+  build(editor, element) {
+    return new Module(editor, element, this.session.data.authenticated.access_token, this.toast);
+  }
+}
+
+export class Module {
+  constructor(editor, element, token, toast) {
+    Object.assign(this, { editor, element, token, toast });
```

Comment:
same comment as above even if it's verbose

Reply:
This is typically where I'd rather have the assign 🤷

Reply:
hmm... true. actually both works so let's keep it like this. (@JulienVannier66 don't take my comments into account)

Reply:
Ok ok